pages/api/create_embedded_sessions.ts

```diff
@@ -114,6 +115,59 @@ const getSessionInvoiceWithFileConnectSessionTokenData = async (
   }
 }
 
+const getSessionInvoiceRelaxedConnectSessionTokenData = ({
+  platformBusinessId,
+  invoices,
+}: CreateEmbeddedSessionPayload): ConnectSessionTokenData => {
+  return {
+    platformBusinessId,
+    data: {
+      component: 'SESSION_INVOICE_FLOW_RELAXED',
+      invoices: invoices.map((invoice) => {
+        const sessionInvoice: SessionInvoice = {
+          payorType: invoice.payorType,
+          platformInvoiceId: invoice.id,
+          platformInvoiceNumber: invoice.invoiceNumber,
+          invoiceAmountCents: getInvoiceTotalCents(invoice, true),
+          invoiceIssuedDate: invoice.createdAtIsoDate,
+          invoiceDueDate: invoice.dueDateIsoDate as string,
```

Comment:
Do you need the casting here?

Reply:
i can remove it. the whole object is being casted to any anyway